server/service/microsoft_mdm.go

```diff
@@ -1004,6 +1009,22 @@ func (svc *Service) storeWindowsMDMEnrolledDevice(ctx context.Context, secTokenM
 		return err
 	}
 
+	// note: computer name is coming provisorily in place of `host_serial`
+	// until we get other bits of the MDM protocol in place. This will be
+	// updated to actually use the serial number in the future.
+	err = svc.ds.NewActivity(ctx, nil, &fleet.ActivityTypeMDMEnrolled{
+		HostSerial:  reqDeviceName,
```

Comment:
@roperzh should we use HostSerial or HostDisplayName here? Disclaimer: I'm not familiar with activity feed so this might be a dumb question

Reply:
I think you're right, that's a way better fit! I pushed a fix